examples/render-helm-chart-crds/.expected/exec.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+kpt fn eval --image-pull-policy ifNotPresent --image gcr.io/kpt-fn/render-helm-chart:unstable --network -- \
```

Comment:
We should use `--image-pull-policy never` since we always build the images before testing.